test/controllers/business_users_controller_test.rb

```diff
@@ -0,0 +1,72 @@
+  require 'test_helper'
+
+class BusinessUsersControllerTest < ActionDispatch::IntegrationTest
+  def setup
+    @business       = businesses(:one)
+    @admin_user     = users(:admin_user)
+    @not_admin_user = users(:not_admin_user)
+  end
+
+	test 'should redirect to login' do
+		get business_business_users_url(@business)
+		assert_response :redirect
+	end
```

Comment:
I'm guessing you copied some code that had "tabs" in it.  Doesn't really matter, just mentioning it.